eng/pipelines/tsp-spec-sync.yml

```diff
@@ -0,0 +1,57 @@
+trigger: none
+
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    oneESTemplateTag: ${{ parameters.oneESTemplateTag }}
+    stages:
+      - stage: TspSpecSync
+        jobs:
+          - job: TspSpecSync
+            displayName: 'TSP Spec Sync Job'
+            pool:
+              name: 'azsdk-pool'
+              image: 'ubuntu-24.04'
+              os: 'linux'
+            steps:
+              - task: UsePythonVersion@0
+                displayName: 'Set up Python'
+                inputs:
+                  versionSpec: '3.10'
+                  addToPath: true
+
+              - script: |
+                  npm install -g @azure-tools/typespec-client-generator-cli@latest
+                displayName: 'Install tsp-client'
+
+              - script: |
+                  git config --global user.name "ADO Pipeline"
+                  git config --global user.email "ado_pipeline@microsoft.com"
```

Comment:
Is this a real user account? Should this instead use our azure-sdk bot? I think I left a comment in your other PR I think the commit and push could happen inside of the create-pull-request.yml template.